src/mondefense.cpp

```diff
@@ -56,9 +56,10 @@ void mdefense::zapback( monster &m, Creature *const source,
     if( const player *const foe = dynamic_cast<player *>( source ) ) {
         // Players/NPCs can avoid the shock if they wear non-conductive gear on their hands
         for( const item &i : foe->worn ) {
-            if( ( i.covers( bodypart_id( "hand_l" ) ) || i.covers( bodypart_id( "hand_r" ) ) ) &&
-                !i.conductive() && ( i.get_coverage( bodypart_id( "hand_l" ) ) >= 95 ) ||
-                i.get_coverage( bodypart_id( "hand_r" ) ) >= 95 ) {
+            if( ( i.covers( bodypart_id( "hand_l" ) ) || i.covers( bodypart_id( "hand_r" ) ) )
+                && !i.conductive()
+                && ( ( i.get_coverage( bodypart_id( "hand_l" ) ) >= 95 ) ||
+                     i.get_coverage( bodypart_id( "hand_r" ) ) >= 95 ) ) {
```

Comment:
Do you even need to test `( i.covers( bodypart_id( "hand_l" ) ) || i.covers( bodypart_id( "hand_r" ) ) )`?  Isn't that covered by the later check that `( ( i.get_coverage( bodypart_id( "hand_l" ) ) >= 95 ) || i.get_coverage( bodypart_id( "hand_r" ) ) >= 95 )`?

Reply:
Nope, I modified this code to work so didn't want to change the logic, but now that you mention it, it is a superfluous check.